enhancements/aws-pod-identity.md

```diff
@@ -0,0 +1,442 @@
+---
+title: aws-pod-identity 
+authors:
+  - "@sjenning"
+reviewers:
+  - "@derekwaynecarr"
+  - "@smarterclayton"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2020-03-26
+last-updated: 2020-03-26
+status: provisional
+see-also:
+  - ""  
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Support Native (IAM-based) Pod Identity on AWS
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+When deployed on AWS, Kubernetes can use pod identity managed by IAM.  AWS has a
+[webhook](https://github.com/aws/amazon-eks-pod-identity-webhook/) they maintain
+for EKS that does enables this ability.  This enhancement integrates the same
+functionality into OpenShift.
+
+The `cloud-credential-operator` will be responsible for managing this new
```

Comment:
- Will this be used by core cluster operators like machine-api, ingress, registry, kube-controller-manager ?
- Or is this only for user workloads.

clarifying that would be nice

Reply:
i covered this in the non-goals section below